add excel sl integration (#5609)
fishtown-analytics/docs.getdbt.com: website/snippets/_sl-partner-links.md

    link="/docs/cloud-integrations/semantic-layer/tableau"
    body="Learn how to connect to Tableau for querying metrics and collaborating with your team."
    icon="tableau-software"/>
  
  <Card
    title="Google Sheets"
    link="/docs/cloud-integrations/semantic-layer/gsheets"
    body="Discover how to connect to Google Sheets for querying metrics and collaborating with your team."
    icon="google-sheets-logo-icon"/>

  <Card

>     title="Microsoft Excel 365 (beta)"

Author: ```suggestion
    title="Microsoft Excel 365 (preview)"
```

>     title="Microsoft Excel 365 (preview)"

Author: ```suggestion
    title="Microsoft Excel (preview)"
```

    link="/docs/cloud-integrations/semantic-layer/excel"

>     body="Connect to Microsoft Excel 365 to query metrics and collaborate with your team."

Author: ```suggestion
    body="Connect to Microsoft Excel to query metrics and collaborate with your team. Available for Excel Desktop or Excel Online."
```

Author: added it here too @rpourzand

>     body="Unlock how to connect to Microsoft Excel 365 to query metrics and collaborate with your team."

Author: ```suggestion
    body="Connect to Microsoft Excel 365 to query metrics and collaborate with your team."
```
